### PR TITLE
Add rpm bugfix bump 8.3.0 --> 8.3.1

### LINKED ISSUE #663: RPM dependency issue with latest agent v9.3.0
#### Agent Version
f5-openstack-agent-9.3.0-1.el7.noarch.rpm

#### Operating System
RHEL 7.3

#### OpenStack Release
RHEL OSP v9 (mitaka)

#### Description
SDK RPM fails to install due to dependency issue.

[root@isft502 9.3.0]# rpm -qa | grep six
python-six-1.9.0-2.el7.noarch

[root@isft502 9.3.0]# rpm -ivh f5-icontrol-rest-1.3.0-1.el7.noarch.rpm
Preparing...                          ################################# [100%]
        package f5-icontrol-rest-1.3.0-1.noarch is already installed

[root@isft502 9.3.0]# rpm -ivh f5-sdk-2.3.1-1.el7.noarch.rpm
error: Failed dependencies:
        f5-icontrol-rest<2.0.0 is needed by f5-sdk-2.3.1-1.noarch
        f5-icontrol-rest>=1.3.0 is needed by f5-sdk-2.3.1-1.noarch
        six<2.0.0 is needed by f5-sdk-2.3.1-1.noarch
        six>=1.9.0 is needed by f5-sdk-2.3.1-1.noarch

Please fix RPM issue.  Thanks!

### PR DESCRIPTION
**Pull request by zancas**
Issues:
Fixes #663

Problem: An sdk issue was preventing RPM installs.

Analysis: See the SDK issue:

https://github.com/F5Networks/f5-common-python/issues/1116

Tests: "SAP" tests were run manually in my bbot-sandbox.

### CODE REVIEW

File: f5_openstack_agent/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "8.3.0"
+__version__ = "8.3.1"
```
Comment (pjbreaux) [inline f5_openstack_agent/__init__.py]: Can you change the version to 8.3.1.b1

**Comment by pjbreaux** [comment]
Once we finish running tests against this code, I will approve.

**Comment by pjbreaux** [comment]
Ran the tests and the results look good. Merging.

**Comment by zancas** [comment]
+1

**Pull request merged**